config/chime_psr_verification.yaml

```diff
@@ -245,18 +245,18 @@ gpu:
     kotekan_stage: hsaProcess
     gpu_id: 0
     updatable_config:
-      gain_psr: /updatable_config/pulsar_gain
-      psr_pt: /updatable_config/pulsar_pointing
+      gain_tracking: /updatable_config/pulsar_gain
+      tracking_pt: /updatable_config/pulsar_pointing
```

Comment:
We should also update the names for the endpoints since we do that in the other configs.